tsdb/head.go

```diff
@@ -652,7 +652,7 @@ func (h *Head) Init(minValidTime int64) error {
 		}
 		// If this fails, data will be recovered from WAL.
 		// Hence we wont lose any data (given WAL is not corrupt).
-		h.removeCorruptedMmappedChunks(err)
+		mmappedChunks = h.removeCorruptedMmappedChunks(err)
```

Comment:
The main fix is here as far as I can see (:

Reply:
While this is a needed fix, it is fine if we missed updating this map, as samples will be recovered from the WAL if no m-mapped chunks.